.github/workflows/deploy-release.yml

```diff
@@ -0,0 +1,38 @@
+# On Tag: Deploy to GitHub Package Registry and create a Release
+name: "Deploy Release"
+
+on:
+  push:
+    tags:
+      - "v**"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```

Comment:
nit: I don’t know what’s the best practice for deploying in Javaland. Isn’t 1.8 really old now, or do they deploy on the oldest supported SDK?

Reply:
I think we could actually go with a newer version now, I think 10 is the new LTS release? From my sparse knowledge now I think java 1.8 is so popular because (1) it's the last release from oracle without the new license bs and (2) it's an LTS release that is now superseeded, but still stable and popular like Windows7.

Reply:
Turns out 1.8 actually is the longer lasting LTS release over Java 11.
![JavaVersions@2x](https://user-images.githubusercontent.com/7768249/82790005-982d0d00-9e6b-11ea-9193-2861496d4e2d.jpg)

Reply:
Oh, I see. I'm fine with anything that's OK for Java programmers.